Index: busybox/networking/udhcp/dhcpc.c
===================================================================
--- busybox.orig/networking/udhcp/dhcpc.c	2013-03-02 08:50:32.414392942 +0900
+++ busybox/networking/udhcp/dhcpc.c	2013-03-02 08:51:03.031321879 +0900
@@ -46,6 +46,7 @@
 	[OPTION_IP_PAIR         ] = sizeof("255.255.255.255 ") * 2,
 	[OPTION_STATIC_ROUTES   ] = sizeof("255.255.255.255/32 255.255.255.255 "),
 	[OPTION_STRING          ] = 1,
+	[OPTION_STRING_HOST     ] = 1,
 #if ENABLE_FEATURE_UDHCP_RFC3397
 	[OPTION_DNS_STRING      ] = 1, /* unused */
 	/* Hmmm, this severely overestimates size if SIP_SERVERS option
