Index: busybox-1.17.1/scripts/gen_build_files.sh
===================================================================
--- busybox-1.17.1.orig/scripts/gen_build_files.sh	2010-08-01 07:28:13.000000000 +0200
+++ busybox-1.17.1/scripts/gen_build_files.sh	2010-08-01 07:31:03.000000000 +0200
@@ -48,7 +48,7 @@
 fi
 
 # (Re)generate */Kbuild and */Config.in
-{ cd -- "$srctree" && find -type d; } | while read -r d; do
+{ cd -- "$srctree" && find -name .\?\* -prune -or -type d -print; } | while read -r d; do
 	d="${d#./}"
 
 	src="$srctree/$d/Kbuild.src"
